Assets/ConnectApp/Components/ChannelMessenger.cs

```diff
@@ -390,6 +393,7 @@ public class ImageMessage : StatefulWidget {
         }
 
         public readonly string url;
+        public readonly byte[] data;
```

Comment:
is there any other solution except storing all the bytes in a Widget ? just wonder the memory cost and efficiency overhead ?